desktop/index.html

```diff
@@ -7,7 +7,7 @@
 
     <meta name="theme-color" content="#000000" />
     <link rel="manifest" href="/manifest.json" />
-    <link rel="shortcut icon" href="/favicon.ico" />
+    <link rel="shortcut icon" href="./assets/icon.ico" />
```

Comment:
If this works on both desktop and web then great. I am a bit worried that it wont work on desktop.

At some point it looks like `favicon.ico` in the `/desktop/public` directory got renamed to `/desktop/public/icon.ico`. 

If ^ doesn't work for both versions then I would try `/icon.ico`

Reply:
Works in both. I'm assuming the build just uses this for the app icon? In which case it works